lib/machinery_helper.rb

```diff
@@ -48,15 +48,15 @@ def can_help?
   end
 
   def inject_helper
-    @system.inject_file(local_helper_path, "/root/")
+    @system.inject_file(local_helper_path, "~")
```

Comment:
I would use a variable for this instead so it can be changed in the future via an option or something.